neuralprophet/forecaster.py

```diff
@@ -3012,3 +3025,30 @@ def _reshape_raw_predictions_to_forecst_df(self, df, predicted, components):
                         yhat_df = pd.Series(yhat, name=comp).set_axis(df_forecast.index)
                         df_forecast = pd.concat([df_forecast, yhat_df], axis=1, ignore_index=False)
         return df_forecast
+
+    def conformalize(self, df_cal, alpha, method="naive", plotting_backend="default"):
+        """Apply a given conformal prediction technique to get the uncertainty prediction intervals (or q-hats).
+
+        Parameters
+        ----------
+            df_cal : pd.DataFrame
+                calibration dataframe
+            alpha : float
+                user-specified significance level of the prediction interval
```

Comment:
What happens if quantiles in QR were already set? Shouldn't it then be using the same?
We could even make it optional, automatically retrieving the QR quantiles and fail if they are not set, non-symmetric or more than two?

Reply:
Not necessarily, you can have `alpha` that is different from QR.

It is possible, by default, to retrieve the QR quantiles, check for symmetry, and automatically alpha, but that can be done as a separate PR.